src/components/ui/menu/menuGroup.component.tsx

```diff
@@ -114,9 +121,10 @@ export class MenuGroup extends React.Component<MenuGroupProps, State> {
 
   private onSubmenuMeasure = (frame: Frame): void => {
     this.setState({ submenuHeight: frame.size.height });
-    if (this.props.initialExpanded) {
+    if (this.initialExpanded) {
```

Comment:
I guess it will have a similar issue. If the menu should stay expanded after subsequent measure it won't work.

Reply:
@whitestranger7 could you check?